xls2xml/xls2xml/tsv2xml.py

```diff
@@ -0,0 +1,69 @@
+"""
+This scripts validates a TSV file with a given schema and convert valid data into XML file
+"""
+# pylint: disable=C0103
+import argparse
+from lxml import etree
+from TSVReader import TSVReader
+from MetadataValidator import MetadataValidator
+from utils import header_to_xml_tag
+
+arg_parser = argparse.ArgumentParser(
```

Comment:
It is usually recommended to decouple the interface code from the rest of the flow, in case it needs to be integrated with other kind of interface. We can leave that for another ticket though.

Reply:
I agree if it is what you meant to separate the conversion logic out to an individual module/function. That indeed makes it easier to reuse and test. This was intended to be a simple script. But yes that is definitely one issue we can improve on.